db/models.py

```diff
@@ -42,11 +51,100 @@ def __str__(self) -> str:
 class MovieSession(models.Model):
     show_time = models.DateTimeField()
     cinema_hall = models.ForeignKey(
-        to=CinemaHall, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=CinemaHall, on_delete=models.CASCADE, related_name="cinema_hall"
     )
     movie = models.ForeignKey(
-        to=Movie, on_delete=models.CASCADE, related_name="movie_sessions"
+        to=Movie, on_delete=models.CASCADE, related_name="movie"
     )
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE
+    )
+
+    class Meta:
+        ordering = ["-created_at"]
+
+    def __str__(self) -> str:
+        return f"{self.created_at.strftime('%Y-%m-%d %H:%M:%S')}"
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(
+        "MovieSession",
+        on_delete=models.CASCADE,
+        null=False
+    )
+    order = models.ForeignKey(
+        "Order",
+        on_delete=models.CASCADE,
+        null=False
+    )
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    constraints = [
+        UniqueConstraint(
+            fields=["row", "seat", "movie_session"],
+            name="unique_ticket")]
+
+    def clean(self) -> None:
+        existing_tickets = Ticket.objects.filter(
+            row=self.row,
+            seat=self.seat,
+            movie_session=self.movie_session
+        )
+
+        if existing_tickets.exists():
+            raise ValidationError("Not Unique Value")
+
+        if not (
+                self.row <= self.movie_session.cinema_hall.rows
+        ):
+            raise ValidationError(
+                {
+                    "row": [
+                        f"row number must be in available "
+                        f"range: (1, rows): "
+                        f"(1, {self.movie_session.cinema_hall.rows})"]
+                }
+            )
+        if not (
+                self.seat <= self.movie_session.cinema_hall.seats_in_row
+        ):
+            raise ValidationError(
+                {
+                    "seat": [
+                        f"seat number must be in available "
+                        f"range: (1, seats_in_row): "
+                        f"(1, {self.movie_session.cinema_hall.seats_in_row})"]
```

Comment:
In case you do not have any argument on the same line with an opening bracket, your closing bracket should not have any arguments on the same line as well.

Reply:
I don`t have this string in my code (to=Movie, on_delete=models.CASCADE, related_name="movie_sessions")